src/node.cc

```diff
@@ -4158,11 +4158,73 @@ inline void PlatformInit() {
 #endif  // __POSIX__
 }
 
+extern "C" {
+  void _register_async_wrap(void);
+  void _register_cares_wrap(void);
+  void _register_fs_event_wrap(void);
+  void _register_js_stream(void);
+  void _register_buffer(void);
+  void _register_config(void);
+  void _register_contextify(void);
+  void _register_crypto(void);
+  void _register_fs(void);
+  void _register_http_parser(void);
+  void _register_icu(void);
+  void _register_os(void);
+  void _register_util(void);
+  void _register_v8(void);
+  void _register_zlib(void);
+  void _register_pipe_wrap(void);
+  void _register_process_wrap(void);
+  void _register_signal_wrap(void);
+  void _register_spawn_sync(void);
+  void _register_stream_wrap(void);
+  void _register_tcp_wrap(void);
+  void _register_timer_wrap(void);
+  void _register_tls_wrap(void);
+  void _register_tty_wrap(void);
+  void _register_udp_wrap(void);
+  void _register_uv(void);
+}
 
 void Init(int* argc,
           const char** argv,
           int* exec_argc,
           const char*** exec_argv) {
+  // Manually initialize all the bindings since auto-initializing is stripped
+  // out in static builds.
+  if (!argc && argv) {
+    // We never actually want to call this code, but when these aren't here they
+    // seem to be optimized away.
+    // TODO: Find a better way to make sure they don't disappear.
```

Comment:
I don't understand what causes this to be needed...  How are you configuring spidernode?